config/default.schema.json

```diff
@@ -364,6 +364,11 @@
 			"type": "string",
 			"description": "The password for Merlin login, used for edusharing"
 		},
+		"ES_MERLIN_ENABLED": {
```

Comment:
What is with the Name convention at the project?

Reply:
ES -> EDU_SHARING

Reply:
ES is Edu-Sharing service
Merlin is the feature we're using

Reply:
But name convention don't intresst it.
Name must be `FEATURE_ES_MERLIN_ENABLED`